lib/cubit/lnurl/lnurl_cubit.dart

```diff
@@ -12,7 +15,19 @@ class LnUrlCubit extends Cubit<LnUrlState> {
   final _log = Logger("LnUrlCubit");
   final BreezSDKLiquid _liquidSdk;
 
-  LnUrlCubit(this._liquidSdk) : super(LnUrlState.initial());
+  LnUrlCubit(this._liquidSdk) : super(LnUrlState.initial()) {
+    _initializeLnUrlCubit();
+  }
+
+  void _initializeLnUrlCubit() {
+    late final StreamSubscription streamSubscription;
+    streamSubscription = _liquidSdk.walletInfoStream.listen(
```

Comment:
NIT: Can use first() instead of creating a stream subscription.

Reply:
Done 124bc7f591b34f0b01efc9685e6df23d1c6b0821